src/VisualStudio/Core/Test.Next/Services/LspDiagnosticsTests.cs

```diff
@@ -423,7 +423,7 @@ static DiagnosticData CreateMockDiagnosticDataWithMappedLocation(Document docume
                 var diagnostic = Diagnostic.Create(descriptor, location);
                 return new DiagnosticData(diagnostic.Id,
                     diagnostic.Descriptor.Category,
-                    null,
+                    message: "",
```

Comment:
See earlier question: are we sure we don't have to deal with diagnostics with no message?